FeedStoreChallenge/CoreDataFeedStore.swift

```diff
@@ -29,14 +29,92 @@ public final class CoreDataFeedStore: FeedStore {
 	}
 
 	public func retrieve(completion: @escaping RetrievalCompletion) {
-		fatalError("Must be implemented")
+		let context = self.context
+		context.perform {
```

Comment:
Suggestion:
During the course this way of having a local reference to the `context` to avoid capturing `self` inside of the `context.perform` block is taught.
The same can be achieved using a capture list.
It saves one line and one variable.

```suggestion
		context.perform { [context] in
```